src/applications/mhv-medical-records/reducers/careSummariesAndNotes.js

```diff
@@ -178,7 +180,7 @@ export const convertAdmissionAndDischargeDetails = record => {
     dischargeDate: dischargeDate ? formatDateLong(dischargeDate) : EMPTY_FIELD,
     dateEntered: dateEntered ? formatDateLong(dateEntered) : EMPTY_FIELD,
     admittedBy: getAttending(summary) || EMPTY_FIELD,
```

Comment:
Is it possible to handle `admittedBy` as well?

Reply:
Yes, thanks for noticing.